Pristine-1.3: Error executing "make check" tests on rinad

After doing a install-user-from-scratch, the "make check" command under the rinad folder throws the following error:

```
/home/rinadev/stack/rinad/src/ipcp/plugins/default/test-routing.cc: In constructor ‘FakeIPCProcess::FakeIPCProcess()’:
/home/rinadev/stack/rinad/src/ipcp/plugins/default/test-routing.cc:395:35: error: invalid new-expression of abstract class type ‘FakeRIBDaemon’
   rib_daemon_ = new FakeRIBDaemon();
```

--- a/rinad/src/ipcp/plugins/default/test-routing.cc
+++ b/rinad/src/ipcp/plugins/default/test-routing.cc
@@ -313,6 +313,20 @@ public:
 		(void) remote_id;
 	}
 
+	void generateCDAPResponse(int invoke_id,
+				  rina::CDAPSessionDescriptor * cdapSessDescr,
+				  rina::CDAPMessage::Opcode opcode,
+				  const std::string& obj_class,
+				  const std::string& obj_name,
+				  rina::RIBObjectValue& robject_value) {
+		(void)invoke_id;
+		(void)cdapSessDescr;
+		(void)opcode;
+		(void)obj_class;
+		(void)obj_name;
+		(void)robject_value;
+	};
+
 private:
 	void operationCalled(const std::string& objectClass,
 	                                 const std::string& objectName,

